extensions/scarb-cairo-test/Cargo.toml

```diff
@@ -13,6 +13,7 @@ cairo-lang-filesystem.workspace = true
 cairo-lang-starknet.workspace = true
 cairo-lang-test-runner.workspace = true
 clap.workspace = true
+scarb = { path = "../../scarb" }
```

Comment:
```suggestion
```

We **very very very much** do not want this to happen. Scarb is a huge dependency and pulling it has many consequences, primarily the executable size grows a lot.

Also, we want to reduce usage of Scarb as a crate, not to increase it.